package.json

```diff
@@ -7,6 +7,8 @@
     "shibuya": "dotenv -e .env babel-node src/astar/shibuya.js",
     "rocstar": "dotenv -e .env babel-node src/astar/rocstar.js",
     "shiden": "dotenv -e .env babel-node src/astar/shiden.js",
+    "setup-moonbase-alpha": "dotenv -e .env babel-node src/moonbeam/setup-moonbase-alpha.js",
```

Comment:
"setup-moonbase-alpha" for simplicity should be fine.

Reply:
Is the file `setup-moonbase-alpha.js` still used? If not, delete that file.

Reply:
Deleted